ci/deps/actions-311.yaml

```diff
@@ -0,0 +1,56 @@
+name: pandas-dev
+channels:
+  - conda-forge
+dependencies:
+  - python=3.11
+
+  # build dependencies
+  - versioneer[toml]
+  - cython>=0.29.32
+
+  # test dependencies
+  - pytest>=7.0
+  - pytest-cov
+  - pytest-xdist>=2.2.0
+  - psutil
+  - pytest-asyncio>=0.17
+  - boto3
+
+  # required dependencies
+  - python-dateutil
+  - numpy<1.24
+  - pytz
+
+  # optional dependencies
+  - beautifulsoup4
+  - blosc
+  - bottleneck
+  - brotlipy
+  - fastparquet
+  - fsspec
+  - html5lib
+  - hypothesis
+  - gcsfs
+  - jinja2
+  - lxml
+  - matplotlib>=3.6.1
+  - numexpr
```

Comment:
Is numba commented out somewhere in this file?

Reply:
No just removed it completely, can add it as a comment if you prefer

Reply:
Can you add it back as a comment so we don't forget it? Thanks.

Reply:
Done